microsoft-edge/devtools-guide-chromium/whats-new/2024/09/devtools-129.md

```diff
@@ -0,0 +1,153 @@
+
+Before:
+
+![Old breakpoint icons](./devtools-129-images/old-breakpoints.png)
+
+After:
+
+![New breakpoint icons](./devtools-129-images/new-breakpoints.png)
+
+
+<!-- ------------------------------ -->
+#### Alignment in UI for setting a conditional breakpoint
+
+The UI that's displayed for setting a conditional breakpoint has been fixed to prevent the width of the textbox from growing and causing alignment issues.
+
+Before:
+
+![Old conditional breakpoint dialog](./devtools-129-images/old-dialog.png)
+
+After:
+
+![New conditional breakpoint dialog](./devtools-129-images/new-dialog.png)
+
+
+<!-- ====================================================================== -->
+## Bug fixes in the Elements tool
+
+<!-- Screen readers announce the "Show element in the Elements panel" button, and the context menu opens when right-clicking a selected text node in the DOM tree. -->
+
+
+<!-- ------------------------------ -->
+#### Screen readers announce the "Show element in the Elements panel" button
+
+In the **Layout** tab, screen readers now announce the **Show element in the Elements panel** (![The "Show element in the Elements panel" icon](./devtools-129-images/show-element-icon.png)) button that's next to each element name:
+
+![The "Show element in the Elements panel" button](./devtools-129-images/show-element-button.png)
+
+
+<!-- ------------------------------ -->
+#### Context menu opens when right-click a selected text node in the DOM tree
+
+Previously, the context menu didn't open after double-clicking a text node in the DOM tree to select the text node, and then right-clicking the text node.  This issue has been fixed; the context menu opens:
+
+![Right-click menu for a selected text node in the DOM tree](./devtools-129-images/context-menu-for-dom-text-node.png)
+
+
+<!-- ====================================================================== -->
+## CSS selector stats Copy table has same order of columns and values
+
+<!-- Subtitle: The Copy table function for CSS selector stats copies data with the values aligned with columns. -->
+
+The **Copy table** function has been fixed to correctly copy the data in the order that the columns and values appear.
+
+![Selector stats copy table function](./devtools-129-images/selector-stats-copy.png)
+
+
+<!-- ====================================================================== -->
+## Icons in the Activity Bar are restored
+
+<!-- Subtitle: Icons in the Activity Bar help you quickly identify the tools. -->
+
+Previously, the following tools were missing icons in the Activity Bar:
+* **Changes**
+* **Network conditions**
+* **Network request blocking**
+* **Performance monitor**
+* **Quick source**
+* **Search**
+
+The icons have been restored for these tools.
```

Comment:
restored, or added?  did these tabs previously in past, have icons?
@leahmsft

Reply:
yup, they used to have icons but went missing at some point

Reply:
"restored" is correct, closing